src/pages/404.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { FourOhFour } from 'gatsby-theme-carbon';
+
+const links = [
+  { href: '/get-started/about-carbon', text: 'About Carbon' },
+  { href: '/components/overview', text: 'Components overview' },
+  { href: '/get-started/development', text: 'Get started: development' },
```

Comment:
is this link still WIP?

Reply:
What do you mean?

Reply:
only this link is broken for me in the deploy preview so I thought it was still WIP
![image](https://user-images.githubusercontent.com/8265238/65278436-bfb82280-daf1-11e9-9b85-cad20967b7d4.png)

Reply:
probably needs an `index.mdx`?